### PR TITLE
docs(pt-br): formatter section

### PR DESCRIPTION
**Pull request by danspinola**

### CODE REVIEW

File: website/src/content/docs/pt-br/formatter/index.mdx [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
 
-The language agnostic options supported by Biome are:
+As opções, agnósticas da linguagem, suportadas pelo Biome são:
 
-- indent style (default: `tab`): Use spaces or tabs for indention
-- tab width (default: `2`): The number of spaces per indention level
-- line width (default: `80`): The column width at which Biome wraps code
+- estilo de indentação (padrão: `tab`): Use espaços ou tabulações para a indentação
+- largura da tabulação (padrão: `2`): O número de espaços por nível de indentação
+- comprimento de linha (padrão: `80`): Número de colunas máximo a partir do qual Biome quebra a linha
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/formatter/index.mdx:19]:
```suggestion
- comprimento de linha (padrão: `80`): Largura máxima da coluna a partir do qual Biome quebra a linha
```

File: website/src/content/docs/pt-br/formatter/index.mdx [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
 
-Other formatting options are available for specific languages as well.  See the [configuration](/reference/configuration) options for details.
+Outras opções de formatação também estão disponíveis para linguagens específicas. Consulte as opções de [configuração](/reference/configuration) para obter detalhes.
 
-## Use the formatter with the CLI
+## Use o formatador com comando de linha
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/formatter/index.mdx:23]:
```suggestion
## Use o formatador com a linha de comando
```

File: website/src/content/docs/pt-br/formatter/index.mdx [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
 
 <PackageManagerBiomeCommand command="format --write ./src" />
 
-Use the  `--help` flag to know what are the available options:
+Use a flag `--help` para saber quais são as opções disponíveis:
 
 <PackageManagerBiomeCommand command="format --help" />
 
-Or check the [CLI reference section](/reference/cli#biomeformat).
+Ou confira a [seção de referência dos comandos de linha](/reference/cli#biomeformat).
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/formatter/index.mdx:37]:
```suggestion
Ou confira a [seção de referência da linha de comando]
```

File: website/src/content/docs/pt-br/formatter/index.mdx [hunk 4]
```diff
@@ -89,21 +89,21 @@ const expr =
   ];
 ```
 
-## Differences with Prettier
+## Diferenças com o Prettier
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/formatter/index.mdx:92]: We move this section "Differences with Prettier" to its own page.
Let's remove it from this file

File: website/src/content/docs/pt-br/formatter/option-philosophy.md [hunk 5]
```diff
@@ -1,38 +1,38 @@
...
+Ter uma opinião tão forte sobre a formatação pode parecer severo demais, mas os benefícios rapidamente se tornam claros após a adoção. Todas as discussões sobre onde os espaços devem ir, se uma linha deve ser quebrada, se uma linha deve ser recuada e muitas outras simplesmente *desaparecem*. [Discussões triviais e desgastantes](https://pt.wikipedia.org/wiki/Lei_da_trivialidade) não tiram mais o foco do realmente importa. As revisões de código ficam livres de solicitações de reformatação e debates cíclicos. Basta confiar que o Biome faz o melhor possível para formatar o código de forma limpa, legível e consistente.
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/formatter/option-philosophy.md:12]:
```suggestion
Ter uma opinião tão forte sobre a formatação pode parecer severo demais, mas os benefícios rapidamente se tornam claros após a adoção. Todas as discussões sobre onde os espaços devem ir, se uma linha deve ser quebrada, se uma linha deve ser recuada e muitas outras simplesmente *desaparecem*. [Discussões triviais e desgastantes](https://pt.wikipedia.org/wiki/Lei_da_trivialidade) não tiram mais o foco do que realmente importa. As revisões de código ficam livres de solicitações de reformatação e debates cíclicos. Basta confiar que o Biome faz o melhor possível para formatar o código de forma limpa, legível e consistente.
```

File: website/src/content/docs/pt-br/formatter/option-philosophy.md [hunk 6]
```diff
@@ -1,38 +1,38 @@
...
 
-> …[these] are not the type of options we’re happy to have. They cause a lot of bike-shedding in teams, and we’re sorry for that. Difficult to remove now, these options exist as a historical artifact and should not motivate adding more options (“If *those* options exist, why can’t this one?”).
+> …[essas] não são opções que gostamos de oferecer. Elas causam muitas discussões inúteis nas equipes, e lamentamos por isso. Difíceis de remover agora, essas opções existem como um artefato histórico e não devem motivar a adição de mais opções ("Se *essas* opções existem, por que esta não pode existir também?").
 
-However, when the [Prettier Challenge was announced](https://twitter.com/Vjeux/status/1722733472522142022), Biome decided to accept the challenge, which required implementing all of the configuration options that Prettier had to achieve full compatibility.
+No entanto, quando o [Prettier Challenge foi anunciado](https://twitter.com/Vjeux/status/1722733472522142022), o Biome decidiu aceitar o desafio, o que exigia a implementação de todas as opções de configuração que o Prettier já oferecia para obter compatibilidade total.
```
Comment (victor-teles) [inline website/src/content/docs/pt-br/formatter/option-philosophy.md:28]:
```suggestion
No entanto, quando o [Desafio do Prettier foi anunciado](https://twitter.com/Vjeux/status/1722733472522142022), o Biome decidiu aceitar o desafio, o que exigia a implementação de todas as opções de configuração que o Prettier já oferecia para obter compatibilidade total.
```

**Review by victor-teles — CHANGES_REQUESTED** [review]
@danspinola Thank you for the contribution 🚀 🥇 

Left some small suggestions

File: website/src/content/docs/pt-br/formatter/index.mdx (hunk 1 above)
Comment (danspinola) [reply website/src/content/docs/pt-br/formatter/index.mdx:19]: Isso está errado, não é sobre a largura da coluna, mas sobre o número de colunas. Cada caractér é uma coluna.

File: website/src/content/docs/pt-br/formatter/index.mdx (hunk 4 above)
Comment (danspinola) [reply website/src/content/docs/pt-br/formatter/index.mdx:92]: A heads up would have been appreciated. So you're telling me I translated it for nothing?

Comment (danspinola) [reply website/src/content/docs/pt-br/formatter/index.mdx:92]: A heads up would have been appreciated. So you're telling me I translated it for nothing?

File: website/src/content/docs/pt-br/formatter/option-philosophy.md (hunk 6 above)
Comment (danspinola) [reply website/src/content/docs/pt-br/formatter/option-philosophy.md:28]: It doesn't make sense to translate the name of the competition.

Comment (victor-teles) [reply website/src/content/docs/pt-br/formatter/option-philosophy.md:28]: We translated it in home page, so lets translate here too.

File: website/src/content/docs/pt-br/formatter/index.mdx (hunk 4 above)
Comment (victor-teles) [reply website/src/content/docs/pt-br/formatter/index.mdx]: Apologies for that.

File: website/src/content/docs/pt-br/formatter/index.mdx (hunk 1 above)
Comment (victor-teles) [reply website/src/content/docs/pt-br/formatter/index.mdx:19]: Ok, Do we need to update the english version too?

File: website/src/content/docs/pt-br/formatter/option-philosophy.md (hunk 6 above)
Comment (danspinola) [reply website/src/content/docs/pt-br/formatter/option-philosophy.md:28]: For what it's worth, I still don't think it makes sense to translate the name of the competition, but I won't insist on it.

**Review by victor-teles — APPROVED** [review]

File: website/src/content/docs/pt-br/formatter/index.mdx (hunk 1 above)
Comment (danspinola) [reply website/src/content/docs/pt-br/formatter/index.mdx:19]: I don't know about that. I did not write the original documentation, nor am I part of the core contributors. However, this is _**pretty standard**_ language. When talking about files or text editors. A file has lines and columns. A column is the same as a character. I think "column's width" would be unequivocally about the "width of the column", but just "column width" can be ambiguous and mean either one. What I know is that "largura da coluna", as if a line has a single column, is wrong, unless Biome is trying to redefine a standard concept.

Comment (victor-teles) [reply website/src/content/docs/pt-br/formatter/index.mdx:19]: Too much effort for something so small.

Lets keep your suggestion, thank you for you hard work.

**Pull request merged**